stremio.pro

```diff
@@ -1,6 +1,6 @@
 TEMPLATE = app
 
-VERSION=4.4.120
+VERSION=4.4.122
```

Comment:
is this supposed to be part of it?

Reply:
Perhaps I was testing. I'll remove this part